### PR TITLE
Kilda gui gradle build structure

### PR DESCRIPTION
**Pull request by neeraj890kumar**
-Changed maven build in gui to gradle 
Resolves:#3841

### CODE REVIEW

**Review by gauravchug — APPROVED** [review]

File: src-gui/Makefile [hunk 1]
```diff
@@ -2,34 +2,36 @@ APP := openkilda-gui
 
 rebuild: clean clean-java build
 
-build: clean target/${APP}.jar
+build:  clean-cache build/libs/${APP}.war
```
Comment (sergii-iakovenko) [inline src-gui/Makefile:5]: Why do we need to clean-cache on each build? This makes impossible incremental builds and forces a developer to wait each time. Can we reconsider this?

File: src-gui/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip
```
Comment (sergii-iakovenko) [inline src-gui/gradle/wrapper/gradle-wrapper.properties:3]: Please, upgrade this to the latest.

**Review by sergii-iakovenko — CHANGES_REQUESTED** [review]
What is the goal to migrate GUI project to gradle builds? For the src-java, it was effective incremental / parallel builds. 
But the proposed version of GUI project still requires 2+ minutes to execute "make build" task, even with no changes on the filesystem.

Also, this PR consists of 2 dedicated change sets: 1 - migration to gradle build, 2 - fix windows end of lines. Why they can't be separated (it was requested by the team and communicated already)?

File: src-gui/Makefile (hunk 1 above)
Comment (neeraj890kumar) [reply src-gui/Makefile:5]: I have already created a task for this. #3734.

File: src-gui/gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (neeraj890kumar) [reply src-gui/gradle/wrapper/gradle-wrapper.properties]: I have upgraded gradle to the latest

**Review by sergii-iakovenko — APPROVED** [review]
Reviewed gradle related files. Looks good.

**Pull request merged**